dask/array/tests/test_routines.py

```diff
@@ -1503,6 +1503,32 @@ def test_unravel_index():
         assert_eq(darr.vindex[d_indices], arr[indices])
 
 
+def test_ravel_multi_index():
+    for arr, chunks, kwargs in [
```

Comment:
Can you make this a parameterized test instead?

Reply:
In a pytest way? I'm not used to pytest yet, is there a good example of what you mean?

Reply:
Essentially you just move the loop outside of the test. https://github.com/dask/dask/blob/3a08bb32e736d51a758281a3abc0fc0dbec04efc/dask/array/tests/test_routines.py#L226-L256 for example

Reply:
Should be fixed now.